Sources/Fullscreen/AddressView/AddressCardView.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+protocol AddressCardViewDelegate: class {
+    func addressCardViewDidSelectCopyButton(_ addressCardView: AddressCardView)
+    func addressCardViewDidSelectGetDirectionsButton(_ addressCardView: AddressCardView)
+}
+
+class AddressCardView: UIView {
+    weak var delegate: AddressCardViewDelegate?
```

Comment:
Should probably be made `public`

Reply:
All the AddressCardView is internal use since it's not meant to be used from outside of the framework. It's this little card

<img width="211" alt="Screenshot 2019-07-18 at 14 17 10" src="https://user-images.githubusercontent.com/1088217/61456589-c2b65b80-a966-11e9-8474-819bdcc96044.png">

Reply:
Ah, I see 👍